testing/src/cypress.config.ts

```diff
@@ -34,5 +30,8 @@ export default defineConfig({
     token_url:
       "https://dev.oidc.gov.bc.ca/auth/realms/jxoe2o46/protocol/openid-connect/token",
     update_institution_url: "/api/institutions/institution",
+    studentURL: "https://test-aest-sims.apps.silver.devops.gov.bc.ca/student/login",
```

Comment:
token URL is `dev` and student URL is `test`

Reply:
The original was set against to dev. But we want to run this against test env for the most part. I ll update all all the endpoints to point the test

Reply:
updated. but all these urls will be moved to separate env file.